fe/src/main/java/org/apache/doris/system/HeartbeatMgr.java

```diff
@@ -176,6 +176,7 @@ private boolean handleHbResponse(HeartbeatResponse response, boolean isReplay) {
                     if (hbResponse.getStatus() != HbStatus.OK) {
                         // invalid all connections cached in ClientPool
                         ClientPool.backendPool.clearPool(new TNetworkAddress(be.getHost(), be.getBePort()));
+                        Catalog.getCurrentCatalog().getGlobalTransactionMgr().abortTxnWhenCoordinateBeDown(be.getHost(), 100);
```

Comment:
Why here is a limit 100?

Reply:
because i afraid too many txn cancel will influence the heatbeat function

Reply:
I see. 
Another 2 things:
1. Plz rebase the code, the FE meta version 82 has been taken.
2. in `handleHbResponse()` method, if the parameter `isReplay` is true, means it is a meta log replay thread, so we don't need to process the abort in a replay thread(This is not like `clearPool()`, which should also be done in replay thread).

Reply:
thanks, I see~, and all things is done.